### PR TITLE
docs(create): create new projects always using latest version

### PR DESCRIPTION
**Pull request by iamandrewluca**
> [!WARNING]
> ⚠️  You're running an old version of create-keystone-app, please update to 10.0.2

```
> npm create keystone-app
> npx
> create-keystone-app


✨ You're about to generate a project using Keystone 6 packages.
⚠️  You're running an old version of create-keystone-app, please update to 10.0.2
✔ What directory should create-keystone-app generate your app into? · test-keystone
...
```

### CODE REVIEW

File: docs/next-env.d.ts [hunk 1]
```diff
@@ -2,4 +2,4 @@
 /// <reference types="next/image-types/global" />
 
 // NOTE: This file should not be edited
-// see https://nextjs.org/docs/basic-features/typescript for more information.
+// see https://nextjs.org/docs/app/building-your-application/configuring/typescript for more information.
```
Comment (iamandrewluca) [inline docs/next-env.d.ts:5]: Note: This was auto generated when I started the docs.

File: .gitignore [hunk 2]
```diff
@@ -11,3 +11,5 @@ __generated__
 # system
 .DS_Store
 *.vscode
+
+.pnpm-store
```
Comment (iamandrewluca) [inline .gitignore:15]: Note: This folder gets created when starting vscode in devcontainer.

**Review by dcousens — APPROVED** [review]
interesting, I thought this was implied but I suppose it may depend on cache?

**Comment by iamandrewluca** [comment]
Probably cache, probably the user has the package installed globally

**Pull request merged**